docs/repos/git/review-pull-requests.md

```diff
@@ -160,9 +165,10 @@ For example:
 A---B---F---G   master
 ```
 
-```git diff branch..master``` will retrieve C, D, E, F, G commits. While ```git diff branch...master``` will produce only F, G.
+Pull Requests: ```git diff branch...master``` will produce only F, G commits. 
+Branch Compare: ```git diff branch..master``` will produce C, D, E, F, G commits.
 
-Here are the relevant documentations about the git diff commands: [three-dot-and-two-dot-git-diff-comparisons](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-comparing-branches-in-pull-requests#three-dot-and-two-dot-git-diff-comparisons) & [git diff](https://git-scm.com/docs/git-diff)
+Here are the link for a relevant documentation with more detail: [three-dot-and-two-dot-git-diff-comparisons](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-comparing-branches-in-pull-requests#three-dot-and-two-dot-git-diff-comparisons) and [git diff](https://git-scm.com/docs/git-diff)
```

Comment:
I'd say
"For more details see: [three-dot-and-two-dot-git-diff-comparisons](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-comparing-branches-in-pull-requests#three-dot-and-two-dot-git-diff-comparisons) and [git diff](https://git-scm.com/docs/git-diff)."
instead of
"Here are the link for a relevant documentation with more detail: [three-dot-and-two-dot-git-diff-comparisons](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-comparing-branches-in-pull-requests#three-dot-and-two-dot-git-diff-comparisons) and [git diff](https://git-scm.com/docs/git-diff)"